websites/P/PcPartPicker/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.4",
+  "service": "PcPartPicker",
+  "author": {
+    "name": "वाह्य",
+    "id": "819696394180100106"
+  },
+  "category": "other",
+  "logo": "https://i.imgur.com/4duoR2Q.png",
+  "thumbnail": "https://i.imgur.com/vWF3XN6.png",
+  "version": "1.0.0",
+  "color": "#c3a041",
+  "tags": ["other"],
```

Comment:
please add some tags related to pcs... not just other...

Reply:
Sorry I was confused between tags and category